gtfs/spec/en/reference.md

```diff
@@ -328,7 +328,7 @@ File: **Optional**
 |   |  | | * **1** - Passenger may transfer once. |
 |   |  | | * **2** - Passenger may transfer twice. |
 |   |  | | * **(empty)** - If this field is empty, unlimited transfers are permitted. |
-|  agency_id | ID | Optional | Required for feeds with multiple agencies defined in the agency.txt file. Each fare attribute must specify an agency_id value to indicate which agency the fare applies to. |
+|  agency_id | ID | **Conditionally required** | Required for feeds with multiple agencies defined in the agency.txt file. Each fare attribute must specify an agency_id value to indicate which agency the fare applies to. This field is required for feeds that contain more than one transit agency in [agency.txt](#agencytxt), otherwise it is optional.|
```

Comment:
I think this should remain `Optional`. If all agencies included in the feed have the same fare attributes & rules, they would need to duplicate everything to meet the conditional requirement.

Reply:
@mgilligan my reading of the existing spec is that this field is already required for feeds with multiple agencies. I don't think the proposed changes modify the meaning of the specification, they just make it more clear.

Reply:
The new language is repetitive though - it says almost exactly the same thing twice.

Reply:
@abyrd, whoops, I didn't read the original in the diff.

@antrim, please ignore my comment. :)